src/jobs/codeql.yml

```diff
@@ -71,6 +75,10 @@ steps:
         cmd+=( --format sarif-latest )
         cmd+=( --output=<< parameters.language >>-codeql-report.sarif )
 
+        if [[ ! -z "<< parameters.analyze_params >>" ]]; then
+          cmd+=( << parameters.analyze_params >> )
+        fi
```

Comment:
Is the if encapsulation really necessary, as if don't provide anything it will simply concatenate nothing with the initial command?
(Buuuuuut, we could agree that this may be cleaner. No strong opinion, mostly thinking out loud here)

Reply:
I feel it is safer and the intention is clearer.